### PR TITLE
Converts the component printer to modular fabricator.

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Changes the component printer into a modular fabricator. Improves the category names of the circuit components.

image

image

image

image

image

image

image

image

## Why It's Good For The Game

The old UI was horrible to use and navigate, modular fabricator is much better when you are trying to print stuff and browse stuff.

## Changelog
:cl:
refactor: Circuit component printer is now a modular fabricator and uses the modular fabricator UI.
/:cl:

### CODE REVIEW

File: code/modules/research/designs/wiremod_designs.dm [hunk 1]
```diff
@@ -1,10 +1,35 @@
+#define WIREMOD_CIRCUITRY "Circuitry"
```
Comment (Archanial) [inline code/modules/research/designs/wiremod_designs.dm]: Move this to defines

**Review by Archanial — CHANGES_REQUESTED** [review]

File: code/game/machinery/fabricators/modular_fabricator.dm [hunk 2]
```diff
@@ -60,6 +60,8 @@
 	var/queue_repeating = FALSE
 	//The amount to readd to the queue when processing is done
 	var/stored_item_amount
+	//Minimum construction time per component
+	var/minimum_construction_time = 35
```
Comment (Archanial) [inline code/game/machinery/fabricators/modular_fabricator.dm:64]: Why isn't this define?

**Review by Archanial — APPROVED** [review]
:+1:

File: code/game/machinery/fabricators/modular_fabricator.dm (hunk 2 above)
Comment (ike709) [reply code/game/machinery/fabricators/modular_fabricator.dm:64]: Not everything needs to be a define

Comment (PowerfulBacon) [reply code/game/machinery/fabricators/modular_fabricator.dm:64]: Defines are good for variables that aren't obvious or repeated in multiple places. Like returning 1, 2 or 3 as status indicators of how a proc went

**Pull request merged**